polaris-react/src/components/Modal/tests/Modal.test.tsx

```diff
@@ -220,6 +220,28 @@ describe('<Modal>', () => {
     });
   });
 
+  describe('fullScreen', () => {
+    it('passes fullScreen to Dialog if true', () => {
+      const modal = mountWithApp(
+        <Modal title="foo" fullScreen onClose={jest.fn()} open>
+          <Badge />
+        </Modal>,
+      );
+
+      expect(modal).toContainReactComponent(Dialog, {fullScreen: true});
+    });
+
+    it('does not pass fullScreen to Dialog be default', () => {
```

Comment:
```suggestion
    it('does not pass fullScreen to Dialog by default', () => {
```

Reply:
Oop, thank you, I had some trouble with the changeset and didnt change this back after a reset >.<